.gitignore

```diff
@@ -23,6 +23,3 @@ yarn-error.log*
 
 public/apps/
 public/aragon-ui/
-
-# ignore package-lock files (only use yarn.lock)
-package-lock.json
```

Comment:
Are we moving back to npm? :thinking:

Reply:
Sorry I didn't leave a comment. The intention of this PR is to serve as a new hot-fix branch to unblock developers using the builder plugin.